src/Microsoft.AspNetCore.WebUtilities/FormReader.cs

```diff
@@ -186,7 +263,7 @@ private void Buffer()
         /// <param name="stream">The HTTP form body to parse.</param>
         /// <param name="cancellationToken">The <see cref="CancellationToken"/>.</param>
         /// <returns>The collection containing the parsed HTTP form body.</returns>
-        public static Task<Dictionary<string, StringValues>> ReadFormAsync(Stream stream, CancellationToken cancellationToken = new CancellationToken())
+        public static ValueTask<Dictionary<string, StringValues>> ReadFormAsync(Stream stream, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
What's the impact of putting `ValueTask<>` in a public API - can mere mortals still call this method?